### PR TITLE
TLC/7096-Dylan-LoadingComponentIndividualMessageAttachmentsText

### PR DESCRIPTION
**Pull request by Sparowhawk**
## Description of Change
Updated loading component and message card loading attachments to have Loading your attachment... text

## Screenshots/Video
Toggle: <details><summary>Before/after:</summary>&nbsp;&nbsp;</details>

## Testing
Yarn test

- [x] Tested on iOS 
- [x] Tested on Android 

## Reviewer Validations
Apply string below spinner, referenced in [#3331](https://github.com/department-of-veterans-affairs/va-mobile-app/issues/3331)
Localize the spinner to Attachments area

## PR Checklist

  **Reviewer:** Confirm the items below as you review

- [x] PR is connected to issue(s)
- [x] Tests are included to cover this change (when possible)
- [x] No magic strings (All string unions follow the [Union -> Constant](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/develop/VAMobile/src/constants/common.ts) type pattern)
- [x] No secrets or API keys are checked in
- [x] All imports are absolute (no relative imports)
- [x] New functions and Redux work have proper TSDoc annotations

## For QA

[Run a build for this branch](https://github.com/department-of-veterans-affairs/va-mobile-app/actions/workflows/on_demand_build.yml)

### CODE REVIEW

File: VAMobile/src/components/LoadingComponent/LoadingComponent.tsx [hunk 1]
```diff
@@ -13,6 +13,8 @@ export type LoadingComponentProps = {
   a11yLabel?: string
   /** Param to show the spinner icon only and not the full page with text */
   justTheSpinnerIcon?: boolean
+  /** Param to show the spinner icon only and not the full page with text */
+  noBoxSpinner?: boolean
```
Comment (narin) [inline VAMobile/src/components/LoadingComponent/LoadingComponent.tsx:17]: Nit, but would naming this something like `inlineSpinner` be a little clearer?

**Review by narin — APPROVED** [review]
Naming suggestion but overall LGTM.

File: VAMobile/src/components/LoadingComponent/LoadingComponent.tsx (hunk 1 above)
Comment (Sparowhawk) [reply VAMobile/src/components/LoadingComponent/LoadingComponent.tsx]: updated

**Review by narin — DISMISSED** [review]

**Review by theodur — DISMISSED** [review]

**Review by TKDickson — COMMENTED** [review]
This.... isn't working in the build from this branch. I'll pop a video/link into the issue itself.

**Comment by Sparowhawk** [comment]
@TKDickson fixed, I found the issue

**Review by TKDickson — APPROVED** [review]
Tested again with the fix - that's now working. Once there's FE PR approval again, this is good to merge.

**Review by theodur — APPROVED** [review]

**Review by alexandec — APPROVED** [review]

**Pull request merged**